lib/bibliothecary/parsers/npm.rb

```diff
@@ -50,6 +50,7 @@ def self.parse_package_lock(file_contents)
 
       def self.parse_manifest(file_contents)
         manifest = JSON.parse(file_contents)
+        raise "is a lockfile and not a manifest" if manifest.key?('lockfileVersion')
```

Comment:
do we know anything about how people get into this situation? 

The error could be more explicit like `package.json contains a lockfileVersion field, expecting this only in package-lock.json` but if we knew how this happened maybe we could be even more helpful

Reply:
Not a clue... but they exist in the wild. They're totally malformed files that npm would choke on afaict from a quick try too.